docs/overview.md

```diff
@@ -1,43 +1,50 @@
 # UI Overview
 
-Monokle is layed out like many other desktop applications:
+Monokle is laid out like many other desktop applications:
 
-![Monokle UI](img/monokle-ui.png)
+![Monokle UI](img/monokle-ui-1.4.0.png)
 
 Left to right:
 
-- The vertical toolbar to the far left allows you to switch between File and Cluster mode.
+- The vertical toolbar to the far left allows you to switch between File, Kustomize, Helm and Cluster mode.
```

Comment:
we should remove "and Cluster mode" from here since it was moved to the top header

Reply:
nice catch @devcatalin

Reply:
or actually - this is for 1.4.0 - so it's ok for now... will have to change in the 1.5.0 documentation